CODEOWNERS

```diff
@@ -41,6 +41,8 @@
 
 /Workbooks/MultiCluster/** @microsoft/container-insights-workbook-contributors @microsoft/applicationinsights-devtools
 
+/Workbooks/MySQL*/** @pradeepkintali @sudheeshgh
```

Comment:
instead of adding specific users, add a team in github instead.  (making one only takes a minute and guarantees everyone has the right access to the microsoft org/etc)

it also makes you in control of your teams' things, and people can be added/removed from the team by you instead of ever touching this file again

Reply:
oh, and you should also add a similar line for your gallery file so that your team owns your gallery file too, not just your template content

Reply:
created a new team @microsoft\azuremysql and updated the code owners for both gallery file and workbook files.